apps/condo/domains/ticket/utils/serverSchema/resolveHelpers.js

```diff
@@ -13,6 +14,8 @@ const { COMPLETED_STATUS_TYPE, NEW_OR_REOPENED_STATUS_TYPE } = require('@condo/d
 const { FLAT_UNIT_TYPE, SECTION_SECTION_TYPE, PARKING_UNIT_TYPE, PARKING_SECTION_TYPE } = require('@condo/domains/property/constants/common')
 const { DEFERRED_STATUS_TYPE, DEFAULT_DEFERRED_DAYS } = require('@condo/domains/ticket/constants')
 
+dayjs.extend(duration)
```

Comment:
why you really needed to use dayjs inside the seter side logic? you can use Date.

Reply:
dayjs can work with duration